src/flavors/williams/static/js/views/app-view.js

```diff
@@ -375,20 +374,13 @@ module.exports = AppView.extend({
       self.placeDetailViews[key] = {};
     });
 
-    _.each(this.landmarks, function(value, key) {
```

Comment:
What was this?

Reply:
We removed all the landmarks stuff in an earlier PR-- I must have found a lingering reference to it here and then deleted it.